MapSample/MapSample/Resources/Raw/AboutAssets.txt

```diff
@@ -0,0 +1,15 @@
+﻿Any raw assets you want to be deployed with your application can be placed in
+this directory (and child directories). Deployment of the asset to your application
+is automatically handled by the following `MauiAsset` Build Action within your `.csproj`.
+
+	<MauiAsset Include="Resources\Raw\**" LogicalName="%(RecursiveDir)%(Filename)%(Extension)" />
+
+These files will be deployed with you package and will be accessible using Essentials:
```

Comment:
These files will be deployed with your package and will be accessible using Essentials: